model/gcp/registry.yaml

```diff
@@ -195,3 +195,28 @@ groups:
         brief: >
           The hostname of a GCE instance. This is the full value of the default or [custom hostname](https://cloud.google.com/compute/docs/instances/custom-hostname-vm).
         examples: ['my-host1234.example.com', 'sample-vm.us-west1-b.c.my-project.internal']
+
+  - id: registry.gcp.gen_ai
+    type: attribute_group
+    display_name: Generative AI Attributes
+    brief: >
+      This documents defines attributes for Google generative AI systems.
+    stability: development
+    attributes:
+      - id: gcp.gen_ai.request
+        stability: development
+        type: template[string[]]
+        brief: >
+          Request configuration options named in a Google-specific manner.
```

Comment:
maybe 

```suggestion
          Gen AI operation options, `<key>` being the flattened name of the configuration option passed to the client library.
```
?

Reply:
Done. Applied suggested change to the `brief` attribute.